keyboards/knobgoblin/knobgoblin.c

```diff
@@ -80,24 +80,23 @@ __attribute__((weak)) void oled_task_user(void) {
 
 	switch (get_highest_layer(layer_state)) {
 		case 0:
-			oled_write_P(PSTR(" ONE\n"), false);
+			oled_write_P(PSTR(" NUM\n"), false);
```

Comment:
please revert these changes. 

If needed, copy the whole function and past it in your keymap